services/director-v2/src/simcore_service_director_v2/modules/osparc_variables/_api_auth.py

```diff
@@ -24,14 +23,14 @@ def _cache_key(fct, *_, **kwargs):
 
 
 @cached(ttl=3, key_builder=_cache_key)
-async def _get_or_create_for(
+async def _create_for(
     app: FastAPI,
     *,
     product_name: ProductName,
     user_id: UserID,
 ) -> ApiKeyGet:
     display_name = create_unique_api_name_for(product_name, user_id)
-    return await get_or_create_api_key_and_secret(
+    return await create_api_key(
```

Comment:
MINOR: since all functions are almost called the same, I prefer `_api_auth_rpc.create_api_key(...)`